tests/unit/models/book-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Model | book', function(hooks) {
+  setupTest(hooks);
+
+  test('it exists', function(assert) {
+    let store = this.owner.lookup('service:store');
+    let model = store.createRecord('book', {});
+    assert.ok(model);
+  });
```

Comment:
Please add 2 tests for the **image** property
1. when there is at least one picture
2. when there is no picture